sysutils/rtx/Portfile

```diff
@@ -0,0 +1,336 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        jdxcode rtx 1.35.8 v
+revision            0
+
+description         Runtime Executor (asdf-plugin compatible)
+
+long_description    rtx is a tool for managing programming language and tool versions. \
+                    For example, use this to install a particular version of node.js and ruby for a project. \
+                    Using rtx activate, you can have your shell automatically switch to the correct node and \
+                    ruby versions when you cd into the project's directory1. Other projects on your machine can \
+                    use a different set of versions. \
+                    rtx is inspired by asdf and uses asdf's vast plugin ecosystem under the hood. \
+                    However, it is much faster than asdf and has a more friendly user experience.
+
+categories          sysutils
+platforms           darwin
+license             MIT
+
+maintainers         @MisLink openmaintainer
```

Comment:
we have both a GitHub handle and an (obfuscated e-mail address) for maintainers. Please add the latter one and group them within curly brackets.